### PR TITLE
searchBar props missing : #2878

### PR DESCRIPTION
**Pull request by singhakshita**
fixes the missing props, and recognised if the input box is already blurred,clear,or focused also initialised missing placeholder prop for the issue #2878 

**Did you add tests for your changes?**
no,not needed for now

**If relevant, did you update the documentation?**
no ,they are already present in documentation

**Summary**
placeholder was not available as a state also clear, focused, blurred  are just "on action" events but now they have been added as a state.

**Does this PR introduce a breaking change?**
No

**Other information**
due to some updation changes yarn.lock and package.json has also been changed a bit , these change are irrevelant for this issue.

### CODE REVIEW

**Comment by singhakshita** [comment]
i was just not too sure about the placeholder props, also its not mentioned in main issue definition .so i removed it from the commit , but please once check the first commit also. this include on focus,onclear,on blurr action types missing props additions which is sure and tested by my side.

File: package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "hoist-non-react-statics": "^3.3.2",
     "lodash.isequal": "^4.5.0",
     "opencollective-postinstall": "^2.0.3",
+    "react-native-elements": "^1.0.0-beta8",
```
Comment (flyingcircle) [inline package.json:47]: Don't change the the package.json or yarn.lock.

**Review by flyingcircle — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (singhakshita) [reply package.json:47]: i didn't change it deliberately ,it changed automatically after yarn install . yeah sure  i will undo it

**Comment by singhakshita** [comment]
sorry i had to create an another pull request , as i was unable to undo yarn and json chnages on this one

File: package.json (hunk 1 above)
Comment (arpitBhalla) [reply package.json:47]: do `git add src/searchbar/SearchBar-default.tsx`

**Pull request closed without merging**